skyVegasTests/gamePlayGameClient.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { test, type Page } from "@playwright/test";
+import { launchGame, login, validateConsoleMessages } from "./utilities";
+import * as fs from "fs";
+import { makeSpin } from "./utilities/makeSpinHelper";
+import { startEventListener } from "./utilities/starteventListenerHelper";
+
+test.describe.configure({ mode: "serial" });
```

Comment:
As in another comment, dont think serial is needed here since its a single test

Reply:
Done :)